css/styles.css

```diff
@@ -0,0 +1,105 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap');
+@import url('./colors.css');
+* {
+	margin: 0;
+	font-family: 'Lato', sans-serif;
+	box-sizing: border-box;
+	word-wrap: break-word;
+}
+body{
+    background-color: var(--primary-bg);
+    color: var(--white);
+}
+.navbar{
+    padding: 2rem;
+    color: var(--white);
+    text-shadow: 2px 2px 2px var(--primary-color);
+    text-align: center;
+}
+.heading{
+    font-size: 2rem;
+    margin-bottom: 1rem;
+}
+.container{
+    display: flex;
+    gap: 1rem;
+    justify-content: center;
+    flex-wrap: wrap;
+}
+.quiz{
+    background-color: var(--dark-bg);
+    border-radius: 5px;
+    width: 18rem;
+}
+.quiz--poster{
+    width: 100%;
+    height: 13rem;
+    border-radius: 5px 5px 0 0;
+}
+.quiz--content{
+    padding: 0.5rem 1rem;
+}
+.quiz--heading{
+    font-size: 1.5rem;
+    text-shadow: 2px 2px 2px var(--primary-color);
+}
+.quiz--description{
+    margin-top: 0.7rem;
+}
+.quiz--btn{
+    text-decoration: none;
+    display: inline-block;
+    margin: 1rem 0;
+    padding: 0.5rem 1rem;
+    background-color: transparent;
+    border-radius: 0.3rem;
+    border: none;
+    text-align: center;
+    outline: none;
```

Comment:
use outline: 1px solid transparent instead

Reply:
sure, thanks for the suggestion